slurm/default.sbatch

```diff
@@ -1,10 +1,9 @@
 #!/usr/bin/env bash
-#SBATCH --time=35:00:00
-#SBATCH --partition=cpu-prio
+#SBATCH --time=96:00:00
+#SBATCH --partition=gpu
 #SBATCH --output=output/output-%j.txt
-#SBATCH --cpus-per-task=15
-#SBATCH --mem-per-cpu=1G
+#SBATCH --cpus-per-task=2
+#SBATCH --mem-per-cpu=4G
```

Comment:
partition should default to cpu with 15 cores and 30 GB of memory (total). max time is okay for a default, imho.